views/play.erb

```diff
@@ -1 +1,11 @@
-<%= @player_1_name %>
\ No newline at end of file
+<h1>Welcome, <%= @player_1_name %>!</h1>
+<h2>Please choose your move</h2>
+<form action='/select' method='post'>
+  <input type="radio" id="rock" name="selection" value="Rock">
+  <label for="rock">Rock</label><br>
+  <input type="radio" id="paper" name="selection" value="Paper">
```

Comment:
I really like the way you chose to use radio buttons to allow the user to pick either rock, paper or scissors. I personally used buttons and thought the other 2 obvious choices were going to be forms and dropdowns, but it's nice to see such a variety of different options when I look at other people's code!